### PR TITLE
add not script

### PR DESCRIPTION
**Pull request by hfaulds**
Closes https://github.com/actions/bin/issues/25

Adds a script which can be used before filters to swap matches with non-matches.

### CODE REVIEW

**Comment by hfaulds** [comment]
I added a bunch of lint and test fixes for other filters. Happy to pull it out into a separate PR if wanted but they're so small I don't see a massive need.

File: filter/test/label.bats [hunk 1]
```diff
@@ -12,7 +12,8 @@ export GITHUB_EVENT_PATH="$BATS_TEST_DIRNAME/fixtures/event.json"
 }
 
 @test "label: matches pull request" {
-  export GITHUB_EVENT_PATH="$BATS_TEST_DIRNAME/fixtures/pull_request_event.json" run label urgent
+  export GITHUB_EVENT_PATH="$BATS_TEST_DIRNAME/fixtures/pull_request_event.json"
+	run label urgent
```
Comment (thepwagner) [inline filter/test/label.bats]:
```suggestion
  run label urgent
```

tabs/spaces.

File: filter/test/user.bats [hunk 2]
```diff
@@ -13,6 +13,7 @@ export GITHUB_EVENT_PATH="$BATS_TEST_DIRNAME/fixtures/event.json"
 }
 
 @test "user: matches pr user" {
+  export GITHUB_EVENT_PATH="$BATS_TEST_DIRNAME/fixtures/pull_request_event.json" run label urgent
```
Comment (thepwagner) [inline filter/test/user.bats]:
```suggestion
```

I'm not sure how this snuck in; it seems to duplicate "label: matches pull request"

Comment (hfaulds) [reply filter/test/user.bats]: @thepwagner this test is failing on msater at the moment, we just didn't notice because it was introduced by a fork PR

**Pull request merged**